mono/mini/mini.h

```diff
@@ -2886,4 +2886,66 @@ gboolean MONO_SIG_HANDLER_SIGNATURE (mono_chain_signal);
 MonoGenericContext
 mono_get_generic_context_from_stack_frame (MonoJitInfo *ji, gpointer generic_info);
 
+#if defined(__APPLE__) && defined(__arm64__)
```

Comment:
This file includes `"mini-arch.h"` which then includes the proper header file. I would move these all to mini-arm64.h and in this file only define if not already defined:
```
#ifndef MONO_SCOPE_ENABLE_JIT_WRITE
#define MONO_SCOPE_ENABLE_JIT_WRITE()
#endif
```

Reply:
Ok I'll move it.